app/backends/service-response.ts

```diff
@@ -93,7 +93,7 @@ export function updateJobFields(
     } else if (status) {
       job.updateStatus(status as JobStatus);
     } else if (redirect) {
-      job.addLink({ href: redirect, rel: 'data' });
+      job.addLink(new JobLink({ href: redirect, rel: 'data' }));
```

Comment:
Lazy Patrick would have made `addLink` accept the original ad hoc object representation and converted it into a `JobLink` internally.  However, as noted later, I think we're actually going to want to persist the new `JobLink` separate from the overall job once we get to pagination, so this is probably better.